### PR TITLE
fix(history): update type

### PR DESCRIPTION
**Pull request by markkaylor**
### What does it do?

- Fixes a type

### Why is it needed?

- It breaks everything

### How to test it?

- Strapi should build

### CODE REVIEW

File: packages/core/content-manager/server/src/history/services/utils.ts [hunk 1]
```diff
@@ -9,13 +9,10 @@ import { FIELDS_TO_IGNORE } from '../constants';
  */
 export const getSchemaAttributesDiff = (
   versionSchemaAttributes: CreateHistoryVersion['schema'],
-  contentTypeSchemaAttributes: Schema.Attributes
+  contentTypeSchemaAttributes: Record<string, Schema.Attribute.AnyAttribute>
```
Comment (alexandrebodin) [inline packages/core/content-manager/server/src/history/services/utils.ts:12]: you can do Schema.Attributes<UID.Schema> to have the right type

File: packages/core/content-manager/server/src/history/services/utils.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { difference, omit } from 'lodash/fp';
-import { Schema } from '@strapi/types';
+import { Struct } from '@strapi/types';
```
Comment (alexandrebodin) [inline packages/core/content-manager/server/src/history/services/utils.ts:2]:
```suggestion
import type { Struct } from '@strapi/types';
```

File: packages/core/content-manager/server/src/history/services/utils.ts (hunk 1 above)
Comment (Convly) [reply packages/core/content-manager/server/src/history/services/utils.ts]: Once #19879 is merged, you will be able to use `Schema.Attributes`

File: packages/core/content-manager/server/src/history/services/utils.ts (hunk 2 above)
Comment (markkaylor) [reply packages/core/content-manager/server/src/history/services/utils.ts]: My bad 😞

**Review by alexandrebodin — APPROVED** [review]

**Pull request merged**